doc/conf.py

```diff
@@ -190,11 +190,22 @@
      'Miscellaneous'),
 ]
 
+scrapers = ('matplotlib',)
+try:
+    import mne
+    import pyvista
+    mne.viz.set_3d_backend('pyvistaqt')
+    pyvista.OFF_SCREEN = False
+    scrapers += ('pyvista',)
+except:
+    pass
```

Comment:
It looks like our docs successfully built Nov 15 and VTK released earlier than that, so it's probably not VTK. I would try using our special Brain scraper:

https://github.com/mne-tools/mne-python/blob/main/doc/conf.py#L353

Reply:
Thx a lot @larsoner :)